gammapy/datasets/map.py

```diff
@@ -2886,3 +2865,46 @@ def irf_position_changed(self):
 
         return changed
 
+
+    def norm_only_changed(self):
+        """Parameters changed"""
+        norm_only_changed = False
+        renorm = 1
+        values = self.model.parameters.value
+        if self._computation_cache is not None:
+            names = self.model.parameters.names
+            ind_diff = np.where(self._cached_parameter_previous != values)[0]
+            if len(ind_diff) == 1:
+                idx = ind_diff[0]
+                norm_only_changed = (
+                    names[idx] in ["norm", "amplitude"]
+                    and self._cached_parameter_previous[idx] != 0
+                )
+                if norm_only_changed:
+                    renorm = values[idx] / self._cached_parameter_previous[idx]
+        return norm_only_changed, renorm
+
+    @property
+    def methods_sequence(self):
```

Comment:
Minor: this could be a lazy property: the order never changes during execution...